airflow/README.md

```diff
@@ -0,0 +1,172 @@
+# Agent Check: Airflow
+
+## Overview
+
+The Datadog Agent collects many metrics from Airflow, including those for:
+
+- DAGs (Directed Acyclic Graphs): Number of DAG processes, DAG bag size, etc
+- Tasks: Task failures, successes, killed, etc
+- Pools: Open slots, used slots, etc
+- Executors: Open slots, queued tasks, running tasks, etc
+- and more
```

Comment:
delete this line, it is implied by the word "including" in line 5

Reply:
Sure. Removed.